src/playground-code-editor.ts

```diff
@@ -255,12 +236,12 @@ export class PlaygroundCodeEditor extends LitElement {
         @keydown=${this._onKeyDown}
       >
         ${this._showKeyboardHelp
-          ? html`<div id="keyboardHelpScrim">
+          ? html`<playground-internal-overlay>
```

Comment:
btw, I don't think this text is very descriptive about what's going on or why the popup is showing. For non screen-reader users there appears to be no action to take. Maybe add some descriptive text about how you can now use <tab> to move focus to other parts of the page?

Also, once you've hit escape to trigger the popup, do you have to hit it against the "exit" the editor? Or is it already in that state? Hitting escape again doesn't appear to do anything (in the playground).

Reply:
Filed https://github.com/google/playground-elements/issues/225 with some thoughts, PTAL.